docs/learn/universal-receiver/create-receiver-forwarder.md

```diff
@@ -273,30 +265,136 @@ IERC725X(msg.sender).execute(0, notifier, 0, encodedTx);
 
 As we know from the `// CHECK that the caller is a LSP0 (UniversalProfile)` test, the `msg.sender` is a Universal Profile which extends `ERC725XCore`. We can then explicitly convert `msg.sender` as a ERC725X contract, then call the `execute` function on it. This means that we "run the execute function as the Universal Profile". The parameters are:
 
-- the `operationType` (0 => CALL operation)
-- the `target` (notifier => our LSP7 contract)
-- the `value` (in native token) (0 => nothing is sent)
+- the `operationType` (0 = CALL operation)
+- the `target` (notifier = our LSP7 contract)
+- the `value` (in native token) (0 = nothing is sent)
 - the `data` (our encoded transaction)
 
-### Method 2
+</TabItem>
 
-In Hardhat, copy the `contracts/LSP1URDForwarderMethod1.sol` file to `contracts/LSP1URDForwarderMethod2.sol` (see command below) and change the following:
+  <TabItem value="method2" label="using LSP1 Forwarder as an operator">
 
-```bash
-cp contracts/LSP1URDForwarderMethod1 contracts/LSP1URDForwarderMethod2.sol
-```
+```solidity title="LSP1URDForwarder.sol"
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.11;
+
+// interfaces
+import { IERC725X } from "@erc725/smart-contracts/contracts/interfaces/IERC725X.sol";
```

Comment:
Maybe you should add packages versions somewhere on that page.

Like "This guide has been tested / is working with versions above 14.0 of lsp-smart-contracts"

or something like that